config/environments/development.rb

```diff
@@ -13,7 +13,7 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local       = false #KIRSTIN for Docker
```

Comment:
I have tested it again and ineed mammooc continues to work, even after I have set this line to true and rebuilt everything. I think the problem may have been resolved by setting the DATABASE_URL. 
Before I experienced the behavior, that it did not work, if this parameter was true and I had not tested again.

Under these circumstances, of course we should keep it true. Thank you for testing it on your machine.